src/libs/ApiUtils.js

```diff
@@ -41,6 +42,11 @@ function getApiRoot(request) {
     const shouldUseSecure = lodashGet(request, 'shouldUseSecure', false);
 
     if (shouldUseStagingServer) {
+        if (CONFIG.IS_USING_WEB_PROXY) {
+            return shouldUseSecure
+                ? proxyConfig.STAGING_SECURE
+                : proxyConfig.STAGING;
+        }
```

Comment:
Pointing out that the "staging toggle" would not work in DEV unless we use the WEB PROXY 

Let's talk this through with internal devs and capture a comment if they agree 

If internal devs want to be able to use the staging server without using the web proxy, then they'll have to apply a similar solution as to how the DEFAULT API works for them (with the `.dev` domain)

This question is still relevant: https://github.com/Expensify/App/pull/16432#discussion_r1146660132 - maybe internal devs don't need to use the "staging toggle" at all so we don't have to do much but add a comment

Reply:
Oh I just commented about the same thing. I don't think we really use this setting on dev, but I don't want the app to break if it is on for some reason. If the EXPENSIFY_URL is `expensify.com.dev` when we trigger the switch to use the staging server we should continue to use EXPENSIFY_URL.